src/route.js

```diff
@@ -6,7 +6,9 @@ module.exports = function route(API, routes={}) {
 
     // For each endpoint (create, read, update, delete...)
     return remap(resource, function(options) {
-      return API.ajax.bind(null, API.config, options)
+      let config = Object.assign({}, API.config, options)
+
+      return Object.assign(API.ajax.bind(null, config), { config })
```

Comment:
Ah I'm creating a new object with a `'config'` key that equals `config`. This is confusing... I should just write it as:

```
let action = API.ajax.bind(null, config)
action.config = config
return config